src/api_server/src/request/drive.rs

```diff
@@ -64,21 +64,6 @@ pub(crate) fn parse_patch_drive(
         ));
     }
 
-    // Validate request - we need to have at least one parameter set:
-    // - path_on_host
-    // - rate_limiter
-    if block_device_update_cfg.path_on_host.is_none()
```

Comment:
actually we would still want this check for virtio block. Is this empty request going to be allowed for it now?

Reply:
Yes. Now empty empty block update call is valid, because it is used for vhost-user-block.